src/storage.js

```diff
@@ -114,9 +114,71 @@ const storageService = (provider, connectionString) => {
     };
   };
 
+  const makeAwsStorage = () => {
+    const AWS = require('aws-sdk');
+    const s3 = new AWS.S3();
+
+    const createWriteStream = (bucketName, fileName) => {
```

Comment:
(bucketName, fileName) => throw new Error("createWriteStream is not supported for AWS")
(this function body {} is redundant)

Reply:
It's a syntax error without the `{}`